objects/obj_controller/Mouse_50.gml

```diff
@@ -75,35 +81,55 @@ if (menu=12) and (cooldown<=0) and (penitorium>0){
                 if (obj_ini.role[c,e]="Master of Sanctity") then tek="c";
                 if (obj_ini.role[c,e]="Master of the Apothecarion") then tek="c";
                 if (obj_ini.role[c,e]="Forge Master") then tek="c";
-                if (obj_ini.role[c,e]="Chapter Master"){tek="c";obj_controller.alarm[7]=5;global.defeat=3;}
-
-                // Needs to be based on role
+                if (obj_ini.role[c,e]="Chapter Master"){
+                    tek="c";
+                    obj_controller.alarm[7]=5;
+                    global.defeat=3;
+                }
+                // TODO Needs to be based on role
                 if (tek="m") then obj_controller.marines-=1;
                 if (tek="c") then obj_controller.command-=1;
 
-                obj_ini.race[c,e]=0;obj_ini.loc[c,e]="";obj_ini.name[c,e]="";obj_ini.role[c,e]="";obj_ini.wep1[c,e]="";obj_ini.lid[c,e]=0;
-                obj_ini.wep2[c,e]="";obj_ini.armour[c,e]="";obj_ini.gear[c,e]="";obj_ini.hp[c,e]=100;obj_ini.chaos[c,e]=0;obj_ini.experience[c,e]=0;
-                obj_ini.mobi[c,e]="";obj_ini.age[c,e]=0;obj_ini.spe[c,e]="";obj_ini.god[c,e]=0;
-                diplo_char=c;with(obj_ini){scr_company_order(obj_controller.diplo_char);}re=1;diplo_char=0;
+                obj_ini.race[c,e]=0;
+                obj_ini.loc[c,e]="";
+                obj_ini.name[c,e]="";
+                obj_ini.role[c,e]="";
+                obj_ini.wep1[c,e]="";
+                obj_ini.lid[c,e]=0;
+                obj_ini.wep2[c,e]="";
+                obj_ini.armour[c,e]="";
+                obj_ini.gear[c,e]="";
+                obj_ini.hp[c,e]=100;
+                obj_ini.chaos[c,e]=0;
+                obj_ini.experience[c,e]=0;
+                obj_ini.mobi[c,e]="";
+                obj_ini.age[c,e]=0;
+                obj_ini.spe[c,e]="";
+                obj_ini.god[c,e]=0;
+                diplo_char=c;
```

Comment:
there is now a generic function for this which should be favoured when the inevitable merge conflict emerges